python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -242,10 +242,10 @@ def delete_detection(self, detection_id: str):
         raise Exception(res.text)
 
     @verify_credentials
-    def create_threat_hunt(self, test_id: str, name: str, query: str, threat_hunt_id: str = None):
+    def create_threat_hunt(self, control: Control, test_id: str, name: str, query: str, threat_hunt_id: str = None):
         """ Create a threat hunt """
         body = dict(
-            control=Control.CROWDSTRIKE.name,
+            control=control.value,
```

Comment:
it works both ways, but we usually use the enum names in requests b/c it's more user friendly (not that anyone is really prying, but still)

Reply:
yup changed back to name